### PR TITLE
Fix: configure Sentry denylist

### LINKED ISSUE #1340: Configure a Sentry denylist
Looks like we can configure a `denylist` on `EventScrubber` when calling `sentry_sdk.init`: https://docs.sentry.io/platforms/python/data-management/sensitive-data/

Another more general approach to modifying events is to configure a `before_send` function: https://docs.sentry.io/platforms/python/configuration/filtering/

_Originally posted by @angela-tran in https://github.com/cal-itp/benefits/issues/1334#issuecomment-1490416579_

### PR DESCRIPTION
**Pull request by angela-tran**
Closes #1340 

This PR configures a custom denylist on our `EventScrubber`, with `sub` and `name` as the additional entries.

### CODE REVIEW

**Review by thekaveman — CHANGES_REQUESTED** [review]
This looks great! I'm reading the docs and they say that for this custom `event_scrubber` to work, `send_default_pii` must be `False` -- (and it is by default!)

But I'd like us to be explicit here, would you mind adding that to the `init` and then also inline-link with a comment to that docs page: https://docs.sentry.io/platforms/python/data-management/sensitive-data/#event_scrubber

**Comment by angela-tran** [comment]
Thanks @thekaveman , those were good suggestions. I put the docs link close to the two parameters, but if you think it'd be better somewhere else, let me know!

File: benefits/sentry.py [hunk 1]
```diff
@@ -68,6 +77,9 @@ def configure():
             environment=SENTRY_ENVIRONMENT,
             release=release,
             in_app_include=["benefits"],
+            # https://docs.sentry.io/platforms/python/data-management/sensitive-data/#event_scrubber
```
Comment (thekaveman) [inline benefits/sentry.py:80]:
```suggestion
            # send_default_pii must be False (the default) for a custom EventScrubber/denylist
            # https://docs.sentry.io/platforms/python/data-management/sensitive-data/#event_scrubber
```

**Review by thekaveman — CHANGES_REQUESTED** [review]
Small additional note

**Review by thekaveman — APPROVED** [review]
👍

**Pull request merged**